azkaban-web-server/build.gradle

```diff
@@ -58,6 +58,9 @@ dependencies {
   testCompile('org.hamcrest:hamcrest-all:1.3')
   testCompile('org.mockito:mockito-all:1.10.19')
 
+  // Tests in azkaban-wer-server module need some Dummy classes put in azkaban-common
```

Comment:
Do you mean web server module?

How about just mention that they need some classes defined in common module's tests? This way the comment stays valid even if you add other dependencies.